apps/mymedicare_cb/views.py

```diff
@@ -109,6 +109,11 @@ def callback(request):
         crosswalk.fhir_id = fhir_id
         crosswalk.save()
 
+        logger.info("Beneficiary connected to FHIR: %s" % crosswalk.fhir_id)
+    else:
+        logger.error("Failed to connect Beneficiary "
+                     "to FHIR: %s" % crosswalk.user_id)
```

Comment:
Same here

Reply:
I have left the messages but removed the id information. This will still be a useful message to track in splunk to identify successful activations or failed matches.